File: Java/프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        int p_c = 0 ,y_c = 0;
        s = s.toUpperCase();
        for(int i = 0; i < s.length();i++){
            if((s.charAt(i) + "").equals("P"))  p_c++;
            else if ((s.charAt(i) + "").equals("Y")) y_c++;
        }
        return (p_c == y_c);
    }
}

File: Python3/프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    p_c,y_c = 0,0
    # 모두 소문자로 바꿔버리는거!!
    p_c = s.lower().count('p')
    y_c = s.lower().count('y')
    return True if p_c == y_c else False